mycli/main.py

```diff
@@ -102,7 +102,10 @@ def __init__(self, sqlexecute=None, prompt=None,
 
         # audit log
         if self.logfile is None and 'audit_log' in c['main']:
-            self.logfile = open(os.path.expanduser(c['main']['audit_log']), 'a')
+            try:
+                self.logfile = open(os.path.expanduser(c['main']['audit_log']), 'a')
+            except (IOError, OSError) as e:
+                self.logfile = False
```

Comment:
You might consider adding an error message here as well so the user won't blindly continue without knowing their audit log file wasn't opened.